lib/anemone/page.rb

```diff
@@ -47,6 +48,8 @@ def initialize(url, params = {})
       @response_time = params[:response_time]
       @body = params[:body]
       @error = params[:error]
+      @skip_no_follow = params[:skip_no_follow]
+      @follow_subdomain = params[:follow_subdomain]
```

Comment:
That's correct, I didn't want to make condition ```when follow_subdomain is false then not checking subdomains```. So I used that option has 3 possible value ```true, false, and array```. Thus, the name a bit confusing. Example for Crawler, with 2 options, we would call it like this:
```ruby
c = Crawler.new(:follow_subdomain => true, :domains => ["this.com", "that.com"])
c = Crawler.new(:follow_subdomain => false)
```
Instead, I set the default option to be true unless User specify it to be false or an Array of domains they allow.